pmd-dist/src/test/java/net/sourceforge/pmd/it/BinaryDistributionIT.java

```diff
@@ -47,27 +48,25 @@ class BinaryDistributionIT extends AbstractBinaryDistributionTest {
         "java-11", "java-12", "java-13", "java-14", "java-15",
         "java-16", "java-17", "java-18", "java-19",
         "java-19-preview", "java-20", "java-20-preview",
-        "java-5", "java-6", "java-7",
-        "java-8", "java-9", "jsp-2", "jsp-3", "kotlin-1.6",
-        "kotlin-1.7", "modelica-3.4", "modelica-3.5",
-        "plsql-11g", "plsql-12.1", "plsql-12.2",
-        "plsql-12c_Release_1", "plsql-12c_Release_2",
-        "plsql-18c", "plsql-19c", "plsql-21c", "pom-4.0.0",
-        "scala-2.10", "scala-2.11", "scala-2.12", "scala-2.13",
-        "swift-4.2", "swift-5.0", "swift-5.1", "swift-5.2",
-        "swift-5.3", "swift-5.4", "swift-5.5", "swift-5.6",
-        "swift-5.7", "vf-52", "vf-53", "vf-54", "vf-55", "vf-56",
-        "vf-57", "vm-2.0", "vm-2.1", "vm-2.2", "vm-2.3", "wsdl-1.1",
-        "wsdl-2.0", "xml-1.0", "xml-1.1", "xsl-1.0", "xsl-2.0",
-        "xsl-3.0"
+        "java-5", "java-6", "java-7", "java-8", "java-9", "jsp-2",
```

Comment:
The point of putting the versions into a list is to not change all lines when a single item changes, which I think is what happened here... Maybe we should put each language on its own line or couple of lines as well?

Reply:
one line per entry is probably going to be too much. But not mixing different languages in a single line should suffice.

Reply:
I've reverted my changes to this section - now only "kotlin-1.8" is added.